README.md

```diff
@@ -15,57 +15,14 @@ Please see [pom.xml](https://github.com/codesupport/website-backend/blob/develop
 2. On Unix systems run `./mnvw clean package` and on Windows run `./mvnw.cmd clean package`
 3. Run `java -jar target/api-service-0.0.1-SNAPSHOT.jar`
 
-## Structure
```

Comment:
Should we move this information to the [wiki](https://github.com/codesupport/website-backend/wiki) or is it not worth having at all?

Reply:
we can add this to the wiki